public-docs/testing-matrix.md

```diff
@@ -0,0 +1,84 @@
+---
+id: testing-matrix
+title: End to End Testing Matrix
```

Comment:
Just a style preference: `End-to-end testing matrix`

Reply:
Done!